chromium_src/third_party/blink/renderer/core/execution_context/navigator_base.cc

```diff
@@ -28,4 +30,13 @@ String NavigatorBase::userAgent() const {
   return userAgent_ChromiumImpl();
 }
 
+unsigned NavigatorBase::hardwareConcurrency(ScriptState* script_state) const {
+  unsigned hardware_concurrency =
+      NavigatorConcurrentHardware::hardwareConcurrency(script_state);
+
+  probe::ApplyHardwareConcurrencyOverride(
```

Comment:
why don't we just get rid of all of these changes and overrides to pass ScriptState and just override `probe::ApplyHardwareConcurrencyOverride` which already receives all the information we need?

Reply:
`#define ApplyHardwareConcurrencyOverride ApplyBraveHardwareConcurrencyOverride(GetExecutionContext(), hardware_concurrency) probe::ApplyHardwareConcurrencyOverride`

Reply:
Addressed in [5e1dfa1](https://github.com/brave/brave-core/pull/13498/commits/5e1dfa1d52180684969c10f204b26b9e5c0681b0) (cc: @pilgrim-brave)